kreuzberg/_extractors/_image.py

```diff
@@ -26,33 +27,17 @@
 class ImageExtractor(Extractor):
     SUPPORTED_MIME_TYPES: ClassVar[set[str]] = IMAGE_MIME_TYPES
 
-    IMAGE_MIME_TYPE_EXT_MAP: ClassVar[Mapping[str, str]] = {
-        "image/bmp": "bmp",
-        "image/x-bmp": "bmp",
-        "image/x-ms-bmp": "bmp",
-        "image/gif": "gif",
-        "image/jpeg": "jpg",
-        "image/pjpeg": "jpg",
-        "image/png": "png",
-        "image/tiff": "tiff",
-        "image/x-tiff": "tiff",
-        "image/jp2": "jp2",
-        "image/jpx": "jpx",
-        "image/jpm": "jpm",
-        "image/mj2": "mj2",
-        "image/webp": "webp",
-        "image/x-portable-anymap": "pnm",
-        "image/x-portable-bitmap": "pbm",
-        "image/x-portable-graymap": "pgm",
-        "image/x-portable-pixmap": "ppm",
-    }
+    IMAGE_MIME_TYPE_EXT_MAP: ClassVar[Mapping[str, str]] = IMAGE_MIME_TO_EXT
```

Comment:
This was moved into a constant out of this file. Remove the class method and use the constant directly